src/main/resources/META-INF/services/za.co.absa.abris.avro.sql.SchemaConverter

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright 2022 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+za.co.absa.abris.avro.sql.DefaultSchemaConverter
```

Comment:
Is this file needed? Since the default is not loaded via ServiceLoader?

Reply:
No, it's not needed. I put it there for consistency, e.g. if someone for some reason wants to configure the abris config with `.withSchemaConverter("default")`. But it's debatable. If someone really wants to do that, he could add that configuration file himself.